hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -302,6 +302,99 @@ public void testUpserts() throws Exception {
         HoodieClientTestUtils.readSince(basePath, sqlContext, timeline, "001").count());
   }
 
+  @Test
```

Comment:
we should unit test the code that cleans up temporary files? Can we add a couple tests around this, since below just seems to be introducing a new test with the params alone set differently? 

IIUC all you want here is to set the two parameters and run one of the tests? lets save some code (from an already huge file) and reuse one of the test methods, by passing in these two parameters/

Reply:
@vinothchandar Unit test added as `testCleanTemporaryDataFiles`